### PR TITLE
Benchmark thrift

### PR DESCRIPTION
**Pull request by fedj**

### CODE REVIEW

File: zipkin4net/Criteo.Profiling.Tracing.Benchmark/README.md [hunk 1]
```diff
@@ -9,8 +9,17 @@ dotnet cli version=1.0.0-preview2-003131
...
 
 ```
- |                              Method |      Mean |    StdDev |
- |------------------------------------ |---------- |---------- |
- |  writeLocalSpanJSONWithCustomParser | 2.2808 us | 0.0159 us |
- | writeClientSpanJSONWithCustomParser | 2.2924 us | 0.0443 us |
- | writeServerSpanJSONWithCustomParser | 2.2982 us | 0.0818 us |
+ |                                                  Method |      Mean |    StdDev |
+ |-------------------------------------------------------- |---------- |---------- |
+ |                      writeLocalSpanJSONWithCustomParser | 1.4042 us | 0.0165 us |
```
Comment (codefromthecrypt) [inline zipkin4net/Criteo.Profiling.Tracing.Benchmark/README.md]: hmm should I be seeing thrift here?

Comment (fedj) [reply zipkin4net/Criteo.Profiling.Tracing.Benchmark/README.md]: You're right, I forgot about rebase though

Comment (fedj) [reply zipkin4net/Criteo.Profiling.Tracing.Benchmark/README.md]: Done

**Comment by codefromthecrypt** [comment]
thx looks very unsurprising relatively speaking. I might take a second look
at the span data as it seems in all cases there should be more difference
between them

**Comment by fedj** [comment]
Do you mean differences between spans themselves or between thrift and json ? I must agree with the first one, length of datas is pretty similar finally

**Comment by codefromthecrypt** [comment]
some nuance of encoding speed will come out when there's more to write. So,
yeah it is about having more realistic spans because that will show
differences that having similar length won't (regardless of same codec or
not)

**Comment by fedj** [comment]
@adriancole I'll do it in another PR to solve #60

**Pull request merged**